data/channels.csv

```diff
@@ -16997,185 +16997,185 @@ KUWBLD3.us,KUWB-LD3,,Defy TV,Nexstar Media,US,US-UT,Bloomington,s/US-UT,eng,ente
+KVCTDT2.us,KVCT-DT2,,Telemundo,SagamoreHill Broadcasting,US,US-TX,Victoria,s/US-TX,spa,general,FALSE,,,,https://www.crossroadstoday.com/,https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQr8c_WMuQx6ScVKpDbeYrzyOno6600xj708zvieuW4rS2YIeK9namdorbVRCKw6WsUAM4&usqp=CAU
+KVCTDT3.us,KVCT-DT3,,CW+,SagamoreHill Broadcasting,US,US-TX,Victoria,s/US-TX,eng,entertainment,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/1/12/This_TV_logo.svg/512px-This_TV_logo.svg.png
+KVCTDT4.us,KVCT-DT4,,H&I,SagamoreHill Broadcasting,US,US-TX,Victoria,s/US-TX,eng,documentary,FALSE,,,,https://www.crossroadstoday.com/,https://pbs.twimg.com/profile_images/1399863507181445120/T25NvQ7V_400x400.jpg
+KVCWDT1.us,KVCW-DT1,,CW,Sinclair Broadcast Group,US,US-NV,Las Vegas,s/US-NV,eng,entertainment,FALSE,1987-08-26,,,http://www.cwlasvegas.com/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/The_CW.svg/512px-The_CW.svg.png
+KVCWDT2.us,KVCW-DT2,,MyNetworkTV,Sinclair Broadcast Group,US,US-NV,Las Vegas,s/US-NV,eng,entertainment,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/3/38/MyNetworkTV_2D_Logo.svg/512px-MyNetworkTV_2D_Logo.svg.png
+KVCWDT3.us,KVCW-DT3,,TBD TV,Sinclair Broadcast Group,US,US-NV,Las Vegas,s/US-NV,eng,entertainment,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/0/0b/TBD_Logo_2019.svg/512px-TBD_Logo_2019.svg.png
+KVCWDT4.us,KVCW-DT4,,The Nest,Sinclair Broadcast Group,US,US-NV,Las Vegas,s/US-NV,eng,entertainment,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/1/12/This_TV_logo.svg/512px-This_TV_logo.svg.png
+KVCWDT5.us,KVCW-DT5,,The Nest,Sinclair Broadcast Group,US,US-NV,Las Vegas,s/US-NV,eng,entertainment,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/9/97/Comet_logo.svg/512px-Comet_logo.svg.png
+KVDADT1.us,KVDA-DT1,,Telemundo,Telemundo Station Group,US,US-TX,San Antonio,s/US-TX,spa,general,FALSE,1989-09-10,,,http://sanantonio.holaciudad.com/,https://upload.wikimedia.org/wikipedia/commons/thumb/6/68/Telemundo_logo_2018.svg/512px-Telemundo_logo_2018.svg.png
+KVDADT2.us,KVDA-DT2,,TeleXitos,Telemundo Station Group,US,US-TX,San Antonio,s/US-TX,spa,entertainment,FALSE,,,,,https://upload.wikimedia.org/wikipedia/en/5/56/TeleXitos_TV.png
+KVDADT3.us,KVDA-DT3,,Cozi TV,Telemundo Station Group,US,US-TX,San Antonio,s/US-TX,eng,entertainment,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/f/f4/Cozi_TV_logo.svg/512px-Cozi_TV_logo.svg.png
+KVDADT4.us,KVDA-DT4,,NBC American Crimes,Telemundo Station Group,US,US-TX,San Antonio,s/US-TX,eng,documentary,FALSE,,,,,https://i.imgur.com/GPoucaV.png
+KVDFCD1.us,KVDF-CD1,,Vision Latina,HC2 Holdings,US,US-TX,San Antonio,s/US-TX,spa,religious,FALSE,1996-09-06,,,https://transition.fcc.gov/fcc-bin/tvq?facid=60464,https://upload.wikimedia.org/wikipedia/commons/a/aa/Azteca_America_2011_logo.png

```

Comment:
Ditto. Been years since AZT America went out. Glad someone's tackling the changes.